apps/system/lockscreen/js/widgets/clock/lockscreen_clock_widget.js

```diff
@@ -50,6 +52,44 @@
     this.logger.debug('Clock updated', now);
   };
 
+	LockScreenClockWidget.prototype.updateAlarm =
+	function() {
+    var self = this;
```

Comment:
Please stop using the old `self` trick while you can use `bind` or arrow function `() => {}`.